src/components/Layout.vue

```diff
@@ -132,10 +133,7 @@ export default class Layout extends Vue {
   }
 
   beforeMount() {
-    if(localStorage.getItem("privatekey")) {
-      // this.setUserIsLoggedIn(true)
-      this.$router.push({ path: '/login' })
-    }
+    if(!this.userIsLoggedIn) this.$router.push({ path: '/login' })
```

Comment:
sorry i don't understand this change. we want to make sure to log people out if they're logged in, and not the opposite

Reply:
Hmm, my reasoning is, If the user refreshes the page `this.userIsLoggedIn` will be false, in which case we want to redirect them to the `/login` page?

Reply:
k, i'll merge and then we test from there